src/Phinx/Db/Action/ChangeColumn.php

```diff
@@ -55,7 +57,7 @@ public function __construct(Table $table, string $columnName, Column $column)
      * @param array<string, mixed> $options Additional options for the column
      * @return static
      */
-    public static function build(Table $table, string $columnName, $type = null, array $options = [])
+    public static function build(Table $table, string $columnName, string|Column|Literal $type, array $options = []): static
```

Comment:
Looks like the docstring on this function was wrong, and similar to above, we should only accept `string|Literal` here as that's what `setType` accepts.